CONTRIBUTING.md

```diff
@@ -56,6 +56,18 @@ The architecture of project is enforced with validation diagrams and will be run
 
 To disable validation for a specific project, set `ValidateLayerDiagram=false`. This is done currently for tests as they are not in the diagram.
 
+Steps :
```

Comment:
Nit: Maybe clarify "Steps to add a new project for validation" or something like that? It's not immediately obvious what these are steps for.